app/models/manageiq/providers/storage_manager.rb

```diff
@@ -9,8 +9,15 @@ class StorageManager < ManageIQ::Providers::BaseManager
     supports_not :smartstate_analysis
     supports_not :block_storage
     supports_not :object_storage
-    supports_not :cloud_object_store_container_create
+    supports_not :cinder_volume_types,                 :description => 'Cinder volume types'
+    supports_not :cloud_object_store_container_create, :description => 'Create Object Store Container'
```

Comment:
This is defined here and also in the base ExtManagementSystem class https://github.com/ManageIQ/manageiq/pull/21046/files#diff-8d286308b75ae7dc8a266753732b0c1709d33293518c479765e34cadf8207910R164  Do we need it in both places?

I see the UI is asking `ExtManagementSystem.any? { |ems| ems.supports?(:cloud_object_store_container_create) }` but to me intuitively it seems like it should live in the `StorageManager`

Reply:
I agree that it should be in the `StorageManager`, but that would need to be coordinated with a change to the UI.  Perhaps in a future set of PRs?

Reply:
The UI code you reference is [here](https://github.com/ManageIQ/manageiq-ui-classic/blob/16406cb713e2646c5928efb287df5c24ec768b39/app/helpers/application_helper/button/cloud_object_store_container_new.rb#L10).  The sole point of it is to figure out whether the `CloudObjectStoreContainerNew` button needs to be enabled on a screen with several providers.  Is there an actual user value to this?  I have always been against actions on list views, as I cannot imagine a user *actually* wanting to do the same operation on multiple items, especially creating a new Object Store Container.  Maybe there are a handful of use cases (VM Start/Stop/Pause) that may be useful for several at a time, but even then I would love to sit behind the user and understand their actual use case.  In either case, we should default to having no operations in list view, and deal with the exceptions.